tests/test_HitTest.cpp

```diff
@@ -0,0 +1,94 @@
+#include <boost/test/unit_test.hpp>
+#include "test_Globals.hpp"
+#include <dynamics/HitTest.hpp>
+#include <engine/GameWorld.hpp>
+#include <dynamics/CollisionInstance.hpp>
+#ifdef _MSC_VER
+#pragma warning(disable : 4305 5033)
+#endif
+#include <btBulletDynamicsCommon.h>
+#include <BulletCollision/CollisionDispatch/btGhostObject.h>
+#ifdef _MSC_VER
+#pragma warning(default : 4305 5033)
+#endif
+
+using std::make_unique;
```

Comment:
`make_unique` and `unique_ptr` are used once in this file. So these lines cause more text :wink: